File: android/src/androidTest/java/net/twisterrob/inventory/android/test/InventoryGlideResetRule.java
package net.twisterrob.inventory.android.test;

import androidx.test.core.app.ApplicationProvider;

import net.twisterrob.android.test.espresso.idle.GlideResetRule;
import net.twisterrob.inventory.android.BuildConfig;
import net.twisterrob.inventory.android.Constants;

class InventoryGlideResetRule extends GlideResetRule {
   @Override protected void before() {
      super.before();
      // recreate internal Glide wrapper
      Constants.Pic.init(ApplicationProvider.getApplicationContext(), BuildConfig.VERSION_NAME);
   }

   @Override protected void after() {
      // No need to clean up after ourselves each test will start with a clean slate,
      // let Glide continue existing after a test.
      //super.after();
   }
}

File: android/src/androidTest/java/net/twisterrob/inventory/android/test/InventoryGlideResetRule.kt
package net.twisterrob.inventory.android.test

import net.twisterrob.android.test.espresso.idle.GlideResetRule
import net.twisterrob.inventory.android.BuildConfig
import net.twisterrob.inventory.android.Constants.Pic

internal class InventoryGlideResetRule : GlideResetRule() {

	override fun before() {
		super.before()
		// Recreate internal Glide wrapper.
		Pic.init(appContext, BuildConfig.VERSION_NAME)
	}

	override fun after() {
		// No need to clean up after ourselves each test will start with a clean slate,
		// let Glide continue existing after a test.
		//super.after();
	}
}
